mkt/extensions/validation.py

```diff
@@ -19,6 +19,8 @@ class ExtensionValidator(object):
     error.
     """
     errors = {
+        'AUTHOR_TOO_LONG': _(
+            u'The `author` property cannot be longer than 255 characters.'),
```

Comment:
I just asked Jacqueline on IRC.  And then confirmed with David Almstrom. If there is a wiki page for the spec it should be (added) there though